about/release_policy.rst

```diff
@@ -57,30 +116,30 @@ for each Godot version:
 .. |unstable| image:: img/unstable.png
 
 **Legend:**
-|supported| Full support -
-|partial| Partial support -
-|eol| No support (end of life) -
+|supported| Full support –
+|partial| Partial support –
+|eol| No support (end of life) –
 |unstable| Development version
 
 Pre-release Godot versions aren't intended to be used in production and are
-provided on a best-effort basis.
+provided for testing purposes only.
 
 .. _doc_release_policy_when_is_next_release_out:
 
 When is the next release out?
 -----------------------------
 
-While Godot contributors aren't working under any deadlines, there's usually a
-major or minor Godot release made available every year. Following this trend,
-this means Godot 4.0 will most likely be released in **2021**.
+While Godot contributors aren't working under any deadlines, we have
+historically had one major or minor release per year, with several maintenance
+updates between each.
 
-Patch releases are made available more frequently, typically every 2-6 months
-while a release is fully supported. Partially supported releases will only have
-new patch releases once an important security or platform support fix has been
-merged.
+Starting with Godot 3.3, we aim to accelerate our development cycles for minor
+releases, so you can expect a new minor release every 3 to 6 months.
 
-.. seealso::
+Maintenance (patch) releases will be released as needed with potentially very
+short development cycles, to provide users of the current stable branch with
+the latest bug fixes for their production needs.
 
-    The `roadmap <https://github.com/godotengine/godot-roadmap>`__ repository
-    documents features that have been agreed upon and may be implemented in future
-    Godot releases.
```

Comment:
Removed the roadmap link as the document is outdated. We can add something back once we define a new roadmap and strive to keep it up to date.